spec/features/authentication/01_sign_up_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+require 'launchy'
+
+feature 'sign up', %Q{
+  As a prospective user
+  I want to create an account
+  So that I can post items and review them
+  } do
+    # ACCEPTANCE CRITERIA
+    # * I must specify a valid email address
+    # * I must specify a password, and confirm that password
+    # * If I do not perform the above, I get an error message
+    # * If I specify valid information, I register my account and am authenticated
```

Comment:
Line is too long. [82/80]<br>Incorrect indentation detected (column 4 instead of 2).